### PR TITLE
i18n: Add Missing Japanese Translations for Legacy Content

### PR DESCRIPTION
**Pull request by niaka3dayo**
## Description

This PR addresses the missing Japanese translations in some legacy content.

## Notes

- This is part of an ongoing effort to complete the Japanese localization for older content.
- I plan to submit multiple PRs to address all missing Japanese translations.
- No changes have been made to the latest content or other languages.

### CODE REVIEW

File: ui/raidboss/data/00-misc/the_masked_carnivale.ts [hunk 1]
```diff
@@ -1353,6 +1391,7 @@ const triggerSet: TriggerSet<Data> = {
           en: 'Away from boss => Stop Everything',
           de: 'Weg vom Boss => Stoppe alles',
           fr: 'Éloignez-vous du boss => Arrêtez tout',
+          ja: 'ボスから離れる => 動かない',
```
Comment (MaikoTan) [inline ui/raidboss/data/00-misc/the_masked_carnivale.ts:1394]: I think this should be 動くな or 止めて, which would be more clear.

**Review by MaikoTan — COMMENTED** [review]
Looks good to me. Just a minor comment!

File: ui/raidboss/data/00-misc/the_masked_carnivale.ts (hunk 1 above)
Comment (niaka3dayo) [reply ui/raidboss/data/00-misc/the_masked_carnivale.ts]: > I think this should be 動くな or 止めて, which would be more clear as this gimmick require you to stop moving, casting and also auto attacking.

Thank you for your comment. I appreciate your suggestion.
You're right that "動くな" (don't move) would be clearer in conveying that this mechanic requires stopping all actions including movement, casting, and auto-attacking.

- Corresponding commit: c35af59c95f022de26d38a63abce45903d078036

**Review by MaikoTan — APPROVED** [review]

**Review by MaikoTan — APPROVED** [review]
LGTM

**Pull request merged**